http.cc

```diff
@@ -77,7 +77,7 @@ int trace_receive_request_header(struct pt_regs *ctx) {
 
 enum { HTTP_EVENT_RECEIVE_REQ, HTTP_EVENT_RECEIVE_REQ_HDR };
 
-struct http_event_t {
+typedef struct st_http_event_t {
```

Comment:
FYI In C++, you can omit `struct`(`class`, `enum`, `union`) keyword without `typedef` unlike c langauge.

```c++
struct Foo {
};

Foo f; // ok

int func(const Foo& f) { // ok
}
```

Reply:
I was also wondering if this is necessary, but there's no harm I suppose.

Reply:
With this change, you cannot use the keyword `struct` before `http_event_t`, which I think is consistent with the h2o style.